.travis.yml

```diff
@@ -17,3 +17,6 @@ install: npm install
 before_script: greenkeeper-lockfile-update
 after_script: greenkeeper-lockfile-upload
 script: npm run test:ci
+cache:
+  directories:
+    - node_modules
```

Comment:
I'm not a TravisCI expert, however, the idea seems pretty sweet. Did you check what's the time benefit that we get from this change?

Reply:
However long it takes to cleanly npm install. It's slow especially with npm 3.x

Reply:
Yeah, I know the idea. But as far as I can see there's no any particular difference:

https://travis-ci.com/mocha-parallel/mocha-parallel-tests/builds/90142875
https://travis-ci.com/mocha-parallel/mocha-parallel-tests/builds/90168111

So does it make sense to use it?

Reply:
If you look at the npm_install step of the builds, you'll see the difference there.